audio/decode/ad_lavc.c

```diff
@@ -105,6 +105,10 @@ static bool init(struct mp_filter *da, struct mp_codec_params *codec,
     ctx->avctx = lavc_context;
     ctx->avframe = av_frame_alloc();
     ctx->avpkt = av_packet_alloc();
+    if (!ctx->avctx || !ctx->avframe || !ctx->avpkt) {
+        MP_ERR(da, "Failed to allocate memory\n");
+        return false;
+    }
```

Comment:
MP_HANDLE_OOM is more appropriate here

Reply:
> MP_HANDLE_OOM is more appropriate here

Done.

Thought about using `MP_HANDLE_OOM` for each allocation instead of chaining them with logical and. It should make it slightly easier to debug/diagnose *in case* it does fail. Worth it, or should I leave it be?

Reply:
not worth it imo